examples/video-only.html

```diff
@@ -44,6 +44,14 @@
             video: true,
             maxLength: 10,
             displayMilliseconds: false,
+            countdown: [
```

Comment:
i'd rather not add it to the default video demo, to keep the example as simple as possible.